agent/src/main/java/com/code_intelligence/jazzer/autofuzz/YourAverageJavaClass.java

```diff
@@ -0,0 +1,227 @@
+// Copyright 2021 Code Intelligence GmbH
```

Comment:
Can you mention the rational behind adding this class?

Reply:
I tested gson and found that we weren't dealing very well with signatures like `deserialize(byte[] data, Class<?> klazz)`. Special casing `Class`, `Method` and `Constructor` to an "interesting" class dramatically increases the coverage here.